pkgs/top-level/aliases.nix

```diff
@@ -178,6 +178,7 @@ mapAliases ({
   google-gflags = gflags; # added 2019-07-25
   googleAuthenticator = google-authenticator; # added 2016-10-16
   grantlee5 = libsForQt5.grantlee;  # added 2015-12-19
+  grimshot = sway-contrib.grimshot; # added 2020-05-18
```

Comment:
I don't understand why we should add this alias. These are AFAIK usually only used for "backwards compatibility" if we rename packages. It also doesn't really help with discoverability as the search will find and show the other attribute (the only exception would be if a user would try to find grimshot via auto-completion).

IMO we should either add `sway-contrib.*` or `grimshot` (XOR not both). And for me the former makes more sense though I find this a very difficult decision...

Reply:
i suppose my motivation is forwards compatibility
i just like `pkgs.grimshot` more than `pkgs.sway-contrib.grimshot`
is there harm in having the alias?
i'm still not sure `grimshot` doesn't at least partially work on non-sway wayland environments

Reply:
One could test in gnome or so. maybe @worldofpeace @colemickens or @jtojnar nows.

Reply:
From a quick look at the code only 2/5 commands (area and screen) should work on other compositors: https://github.com/swaywm/sway/blob/master/contrib/grimshot#L112
So I doubt I'll make sense outside of Sway (especially since both cases are already covered by grim+slurp).

@Mic92 do you have an opinion on the alias? I find this a bit tricky since `grimshot` is part of the Sway repo (in `contrib/`) and not a stand-alone package. See https://github.com/NixOS/nixpkgs/issues/87979.

Reply:
Maybe not expose it in that case.

Reply:
@Mic92 ok, thanks.

@evils After reading [Add a "contrib" directory to your projects](https://drewdevault.com/2020/06/06/Add-a-contrib-directory.html) I feel confirmed that we should not expose the `grimshot` alias (even though the post doesn't really cover that part). Can you drop the alias then?

Reply:
i removed the alias since you feel so strongly and probably have thought this though more than i have